packages/core/styles/components/_toast.scss

```
@@ -135,6 +135,18 @@ ix-toast {
     border-left: 3px solid var(--ix-border);
   }
 
+    > aside[data-intent="gradient"] {
+
+    border-left: 3px solid var(--ix-gradient);
```

Stylelint (17.10.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)